flake.lock

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I'd prefer not including Nix-specific files, I won't be able to maintain them.

Reply:
I removed the `flake.nix`